nixos/tests/systemd-initrd.nix

```diff
@@ -0,0 +1,24 @@
+import ./make-test-python.nix ({ lib, pkgs, ... }: {
+  name = "systemd-initrd";
+
+  machine = { pkgs, ... }: {
+    boot.initrd.systemd.enable = true;
+    fileSystems = lib.mkVMOverride {
+      "/".autoResize = true;
```

Comment:
This is already the case, isn't it?

Reply:
It is not. Currently, stage 1 doesn't have an implementation of auto resizing and formatting. The things we have that do this (like qemu-vm or sd-image) have their own special code for it. In fact, before this PR, setting `fileSystems."/".autoResize = true;` would result in errors because the code we have generating systemd units for resizing didn't handle "/" gracefully.